javascript/javascript1/week1/homework-week1-hossein.js

```diff
@@ -0,0 +1,47 @@
+// Javascript warm up part one
+let myFreeCodeCampUserName="@Hossein1065";
+
+// Age-ify (A future age calculator)
+var yearOfBirth =1986;
+var yearFuture=2025;
+var age=yearFuture-yearOfBirth;
+console.log("you will be "+ age + " years old in "+ yearFuture);
+
+//Goodboy-Oldboy (A dog age calculator)
+var dogYearOfBirth=2015;
+var dogYearFuture=2045;
+var dogYear= (dogYearFuture-dogYearOfBirth)*7;
+var shouldShowResultInDogYears;
+if (shouldShowResultInDogYears =true){
+
+console.log("it is dog years and your dog will be " +dogYear+" years old in "+dogYearFuture);
+}
+else{
+console.log("it is human years and you dog will be " + (dogYear/7) +" years old in "+dogYearFuture)
+}
```

Comment:
It looks fancy! Great job. but I have a question for you, How can I see the dog's age in human year?
Though, **shouldShowResultInDogYears** will always be true unless you specify the opposite.
You can discover that by adding this line 
```suggestion
var shouldShowResultInDogYears;
if (shouldShowResultInDogYears =true){

console.log("it is dog years and your dog will be " +dogYear+" years old in "+dogYearFuture);
  console.log(shouldShowResultInDogYears);

}
else{
console.log("it is human years and you dog will be " + (dogYear/7) +" years old in "+dogYearFuture)
}
```

But I will still not be able to toggle between human years ad dog's years.
Think about a different implementation for that part.